### PR TITLE
Miscellaneous improvements to MathML DOM tests

### LINKED ISSUE #19500: Don't use asynchronous test to check MathMLElement onevent attributes
cc @rwlbuis @bkardell 

In mathml/relations/html5-tree/clipboard-event-handlers.tentative.html
and  mathml/relations/html5-tree/math-global-event-handlers.tentative.html 

@bzbarsky suggested we used the following instead:

    var callbackCalled = false;
    function callback() {
          callbackCalled = true;
    }
    doStuffThatShouldCallCallback();
    assert_true(callbackCalled);

### PR DESCRIPTION
**Pull request by fred-wang**
- Don't use asynchronous test to check MathMLElement onevent attributes.
  closes #19500
- Only test tab-navigation of MathML links when HTML links are tab-navigable.
  https://github.com/mathml-refresh/mathml/issues/152
  https://github.com/Igalia/chromium-dev/issues/50
  https://bugzilla.mozilla.org/show_bug.cgi?id=1571487#c30

### CODE REVIEW

**Comment by fred-wang** [comment]
cc @bzbarsky @emilio

File: mathml/relations/html5-tree/tabindex-002.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
 <script src="/resources/testdriver.js"></script>
 <script src="/resources/testdriver-vendor.js"></script>
 <div id="log"></div>
+<a tabindex="1" href="#link">tabindex(1)-html</a>
```
Comment (bzbarsky) [inline mathml/relations/html5-tree/tabindex-002.html]: Why the tabindex="1" here?  The mtext that's a link doesn't have that...

Comment (fred-wang) [reply mathml/relations/html5-tree/tabindex-002.html]: That was to make the HTML link focused before *all* the other focusable MathML elements in this test. This explicit tabindex="1" does not make the link tab-navigable on macOS. 

However, it's true that we actually only need that HTML link to be focused before the mtext link and removing the tabindex attribute make them more similar. So I'll to remove it.

**Review by rwlbuis — APPROVED** [review]
LGTM.

**Pull request merged**